types/office-js-preview/index.d.ts

```diff
@@ -9744,10 +9744,12 @@ declare namespace Office {
     }
 
     /**
-     * The InternetHeaders object represents properties that are preserved after the message item leaves Exchange and is converted to a MIME message. 
-     * These properties are stored as x-headers in the MIME message.
+     * The InternetHeaders object represents custom internet headers that are preserved after the message item leaves Exchange and is converted to a MIME message. 
```

Comment:
It might be worth mentioning the custom headers are meant to be solely interacted with by your add-in.

Reply:
I'll leave it be for now. Thanks.